packages/website/babel.config.js

```diff
@@ -16,3 +18,11 @@ module.exports = {
     '@babel/transform-runtime',
   ],
 };
+
+// to merge the consumer level babel.config
+// condition required to support dev testing of our website which otherwise throws Configuration cycle detected loading error.
+if (cwd !== __dirname) {
+  babelConfig.extends = `${cwd}/babel.config.js`;
```

Comment:
Can we only set this if the files exists? We can use `existsSync` to check on this before setting it.

Also, can we support a `.babelrc` as well as a `babel.config.js`.